### ISSUE TITLE
Error Running Workflow after upgrading Torch Pack to 2.11.0+cu130

### ISSUE THREAD

**Issue by 4rc4nux** [Tavris1/ComfyUI-Easy-Install#276]
Ezi loads fine without errors but when running my default workflow, a fatal error occurs and the app tries to restart (hangs indefinitely)

Reverting back to 2.10.0+cu130 working fine.

The error occurs when running Ksampler SDXL (Eff.) (Effeciency Nodes)

Log:

[INFO] Requested to load SDXL
[INFO] Model SDXL prepared for dynamic VRAM loading. 4896MB Staged. 722 patches attached. Force pre-loaded 512 weights: 1197 KB.
  0%|                                                                                                                                                                                                                                                               | 0/40 [00:00<?, ?it/s]
  0%|                                                                                                                                                                                                                                   | 0/40 [00:00<?, ?it/s,   Model Initializing ...  ]Fatal Python error: Aborted
 
Stack (most recent call first):
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\sageattention\core.py", line 149 in sageattn
  File "I:\ComfyUI-EZi\ComfyUI\comfy\ldm\modules\attention.py", line 596 in attention_sage
  File "I:\ComfyUI-EZi\ComfyUI\comfy\ldm\modules\attention.py", line 178 in wrapper
  File "I:\ComfyUI-EZi\ComfyUI\comfy\ldm\modules\attention.py", line 856 in forward
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1790 in _call_impl
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1779 in _wrapped_call_impl
  File "I:\ComfyUI-EZi\ComfyUI\comfy\ldm\modules\attention.py", line 962 in forward
  File "I:\ComfyUI-EZi\ComfyUI\custom_nodes\ComfyUI-Easy-Use\py\modules\layer_diffuse\attension_sharing.py", line 252 in forward
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1790 in _call_impl
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1779 in _wrapped_call_impl
  File "I:\ComfyUI-EZi\ComfyUI\comfy\ldm\modules\attention.py", line 1076 in forward
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1790 in _call_impl
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1779 in _wrapped_call_impl
  File "I:\ComfyUI-EZi\ComfyUI\comfy\ldm\modules\diffusionmodules\openaimodel.py", line 56 in forward_timestep_embed
  File "I:\ComfyUI-EZi\ComfyUI\comfy\ldm\modules\diffusionmodules\openaimodel.py", line 880 in _forward
  File "I:\ComfyUI-EZi\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "I:\ComfyUI-EZi\ComfyUI\comfy\ldm\modules\diffusionmodules\openaimodel.py", line 838 in forward
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1790 in _call_impl
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1779 in _wrapped_call_impl
  File "I:\ComfyUI-EZi\ComfyUI\comfy\model_base.py", line 238 in _apply_model
  File "I:\ComfyUI-EZi\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "I:\ComfyUI-EZi\ComfyUI\comfy\model_base.py", line 194 in apply_model
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 334 in _calc_cond_batch
  File "I:\ComfyUI-EZi\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 218 in _calc_cond_batch_outer
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 210 in calc_cond_batch
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 619 in sampling_function
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 1212 in predict_noise
  File "I:\ComfyUI-EZi\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 1209 in outer_predict_noise
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 1202 in __call__
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 639 in __call__
  File "I:\ComfyUI-EZi\ComfyUI\comfy\k_diffusion\sampling.py", line 225 in sample_euler_ancestral
  File "I:\ComfyUI-EZi\python_embeded\Lib\site-packages\torch\utils\_contextlib.py", line 124 in decorate_context
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 999 in sample
  File "I:\ComfyUI-EZi\ComfyUI\custom_nodes\ComfyUI-TiledDiffusion\utils.py", line 34 in KSAMPLER_sample
  File "I:\ComfyUI-EZi\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 1229 in inner_sample
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 1254 in outer_sample
  File "I:\ComfyUI-EZi\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 1316 in sample
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 1334 in sample
  File "I:\ComfyUI-EZi\ComfyUI\comfy\samplers.py", line 1444 in sample
  File "I:\ComfyUI-EZi\ComfyUI\custom_nodes\ComfyUI-TiledDiffusion\utils.py", line 51 in KSampler_sample
  File "I:\ComfyUI-EZi\ComfyUI\comfy\sample.py", line 74 in sample
  File "I:\ComfyUI-EZi\ComfyUI\custom_nodes\comfyui-multigpu\__init__.py", line 508 in sample_with_runtime_device
  File "I:\ComfyUI-EZi\ComfyUI\nodes.py", line 1571 in common_ksampler
  File "I:\ComfyUI-EZi\ComfyUI\nodes.py", line 1641 in sample
  File "I:\ComfyUI-EZi\ComfyUI\custom_nodes\efficiency-nodes-comfyui\efficiency_nodes.py", line 574 in process_latent_image
  File "I:\ComfyUI-EZi\ComfyUI\custom_nodes\efficiency-nodes-comfyui\efficiency_nodes.py", line 741 in sample
  File "I:\ComfyUI-EZi\ComfyUI\custom_nodes\efficiency-nodes-comfyui\efficiency_nodes.py", line 2284 in sample_sdxl
  File "I:\ComfyUI-EZi\ComfyUI\execution.py", line 304 in process_inputs
  File "I:\ComfyUI-EZi\ComfyUI\execution.py", line 316 in _async_map_node_over_list
  File "I:\ComfyUI-EZi\ComfyUI\execution.py", line 342 in get_output_data
  File "I:\ComfyUI-EZi\ComfyUI\execution.py", line 543 in execute
  File "I:\ComfyUI-EZi\ComfyUI\execution.py", line 785 in execute_async
  File "asyncio\events.py", line 88 in _run
  File "asyncio\base_events.py", line 1999 in _run_once
  File "asyncio\base_events.py", line 645 in run_forever
  File "asyncio\windows_events.py", line 322 in run_forever
  File "asyncio\base_events.py", line 678 in run_until_complete
  File "asyncio\runners.py", line 118 in run
  File "asyncio\runners.py", line 195 in run
  File "I:\ComfyUI-EZi\ComfyUI\execution.py", line 725 in execute
  File "I:\ComfyUI-EZi\ComfyUI\main.py", line 359 in prompt_worker
  File "threading.py", line 1012 in run
  File "threading.py", line 1075 in _bootstrap_inner
  File "threading.py", line 1032 in _bootstrap
 
Extension modules: sqlalchemy.cyextension.collections, sqlalchemy.cyextension.immutabledict, sqlalchemy.cyextension.processors, sqlalchemy.cyextension.resultproxy, sqlalchemy.cyextension.util, greenlet._greenlet, markupsafe._speedups, yaml._yaml, PIL._imaging, multidict._multidict, yarl._quoting_c, propcache._helpers_c, aiohttp._http_writer, aiohttp._http_parser, aiohttp._websocket.mask, aiohttp._websocket.reader_c, frozenlist._frozenlist, charset_normalizer.md, charset_normalizer.cd, numpy.core._multiarray_umath, numpy.core._multiarray_tests, numpy.linalg._umath_linalg, numpy.fft._pocketfft_internal, numpy.random._common, numpy.random.bit_generator, numpy.random._bounded_integers, numpy.random._mt19937, numpy.random.mtrand, numpy.random._philox, numpy.random._pcg64, numpy.random._sfc64, numpy.random._generator, torch._C, torch._C._dynamo.autograd_compiler, torch._C._dynamo.eval_frame, torch._C._dynamo.guards, torch._C._dynamo.utils, torch._C._fft, torch._C._linalg, torch._C._nested, torch._C._nn, torch._C._sparse, torch._C._special, psutil._psutil_windows, PIL._imagingft, _cyutility, scipy._cyutility, scipy._lib._ccallback_c, scipy.ndimage._nd_image, scipy.ndimage._rank_filter_1d, scipy.special._ufuncs_cxx, scipy.special._ellip_harm_2, scipy.special._special_ufuncs, scipy.special._gufuncs, scipy.special._ufuncs, scipy.special._specfun, scipy.special._comb, _ni_label, scipy.ndimage._ni_label, regex._regex, sentencepiece._sentencepiece, scipy.integrate._odepack, scipy.integrate._quadpack, scipy.integrate._vode, scipy.integrate._dop, scipy.sparse._sparsetools, _csparsetools, scipy.sparse._csparsetools, scipy.linalg._fblas, scipy.linalg._flapack, scipy.linalg.cython_lapack, scipy.linalg._cythonized_array_utils, scipy.linalg._solve_toeplitz, scipy.linalg._batched_linalg, scipy.linalg._decomp_lu_cython, scipy.linalg._matfuncs_schur_sqrtm, scipy.linalg._matfuncs_expm, scipy.linalg._linalg_pythran, scipy.linalg.cython_blas, scipy.linalg._decomp_update, scipy.sparse.linalg._dsolve._superlu, scipy.sparse.linalg._eigen.arpack._arpacklib, scipy.sparse.linalg._propack, scipy.optimize._group_columns, scipy._lib.messagestream, scipy.optimize._trlib._trlib, scipy.optimize._lbfgsb, _moduleTNC, scipy.optimize._moduleTNC, scipy.optimize._slsqplib, scipy.optimize._minpack, scipy.optimize._lsq.givens_elimination, scipy.optimize._zeros, scipy._lib._uarray._uarray, scipy.linalg._decomp_interpolative, scipy.optimize._bglu_dense, scipy.optimize._lsap, scipy.spatial._ckdtree, scipy.spatial._qhull, scipy.spatial._voronoi, scipy.spatial._hausdorff, scipy.spatial._distance_wrap, scipy.spatial.transform._rotation_cy, scipy.spatial.transform._rigid_transform_cy, scipy.optimize._direct, scipy.interpolate._fitpack, scipy.interpolate._dfitpack, scipy.interpolate._dierckx, scipy.interpolate._ppoly, scipy.interpolate._interpnd, scipy.interpolate._rbfinterp_pythran, scipy.interpolate._rgi_cython, scipy.special.cython_special, scipy.stats._stats, scipy.stats._biasedurn, scipy.stats._stats_pythran, scipy.stats._levy_stable.levyst, scipy.stats._ansari_swilk_statistics, scipy.sparse.csgraph._tools, scipy.sparse.csgraph._shortest_path, scipy.sparse.csgraph._traversal, scipy.sparse.csgraph._min_spanning_tree, scipy.sparse.csgraph._flow, scipy.sparse.csgraph._matching, scipy.sparse.csgraph._reordering, scipy.stats._sobol, scipy.stats._qmc_cy, scipy.stats._rcont.rcont, scipy.stats._qmvnt_cy, av._core, av.logging, av.bytesource, av.buffer, av.audio.format, av.error, av.dictionary, av.container.pyio, av.option, av.descriptor, av.format, av.utils, av.stream, av.container.streams, av.sidedata.motionvectors, av.sidedata.sidedata, av.opaque, av.packet, av.container.input, av.container.output, av.container.core, av.codec.context, av.video.format, av.video.reformatter, av.plane, av.video.plane, av.video.frame, av.video.stream, av.codec.hwaccel, av.codec.codec, av.frame, av.audio.layout, av.audio.plane, av.audio.frame, av.audio.stream, av.filter.link, av.filter.context, av.filter.graph, av.filter.filter, av.filter.loudnorm, av.audio.resampler, av.audio.codeccontext, av.audio.fifo, av.bitstream, av.video.codeccontext, av.subtitles.stream, scipy.signal._sigtools, scipy.signal._max_len_seq_inner, scipy.signal._upfirdn_apply, scipy.signal._spline, scipy.signal._sosfilt, scipy.signal._peak_finding_utils, sklearn.__check_build._check_build, pandas._libs._cyutility, pandas._libs.tslibs.ccalendar, pandas._libs.tslibs.np_datetime, pandas._libs.tslibs.dtypes, pandas._libs.tslibs.base, pandas._libs.tslibs.nattype, pandas._libs.tslibs.timezones, pandas._libs.properties, pandas._libs.tslibs.fields, pandas._libs.tslibs.timedeltas, pandas._libs.tslibs.tzconversion, pandas._libs.tslibs.timestamps, pandas._libs.tslibs.offsets, pandas._libs.tslibs.strptime, pandas._libs.tslibs.parsing, pandas._libs.tslibs.conversion, pandas._libs.tslibs.period, pandas._libs.tslibs.vectorized, pandas._libs.ops_dispatch, pandas._libs.missing, pandas._libs.hashtable, pandas._libs.algos, pandas._libs.interval, pandas._libs.lib, pandas._libs.ops, pandas._libs.hashing, pandas._libs.arrays, pandas._libs.tslib, pandas._libs.sparse, pandas._libs.internals, pandas._libs.indexing, pandas._libs.index, pandas._libs.writers, pandas._libs.join, pandas._libs.window.aggregations, pandas._libs.window.indexers, pandas._libs.reshape, pandas._libs.groupby, pandas._libs.json, pandas._libs.parsers, pandas._libs.testing, sklearn._cyutility, sklearn.utils._isfinite, sklearn.utils.sparsefuncs_fast, sklearn.utils.murmurhash, sklearn.utils._openmp_helpers, sklearn.metrics.cluster._expected_mutual_info_fast, sklearn.metrics._dist_metrics, sklearn.metrics._pairwise_distances_reduction._datasets_pair, sklearn.utils._cython_blas, sklearn.metrics._pairwise_distances_reduction._base, sklearn.metrics._pairwise_distances_reduction._middle_term_computer, sklearn.utils._heap, sklearn.utils._sorting, sklearn.metrics._pairwise_distances_reduction._argkmin, sklearn.metrics._pairwise_distances_reduction._argkmin_classmode, sklearn.utils._vector_sentinel, sklearn.metrics._pairwise_distances_reduction._radius_neighbors, sklearn.metrics._pairwise_distances_reduction._radius_neighbors_classmode, sklearn.metrics._pairwise_fast, sklearn.preprocessing._csr_polynomial_expansion, sklearn.preprocessing._target_encoder_fast, _cffi_backend, google._upb._message, simsimd, stringzilla, insightface.thirdparty.face3d.mesh.cython.mesh_core_cython, kiwisolver._cext, scipy.io.matlab._mio_utils, scipy.io.matlab._streams, scipy.io.matlab._mio5_utils, pycocotools._mask, zstandard.backend_c, ormsgpack.ormsgpack, numba.core.typeconv._typeconv, numba._helperlib, numba._dynfunc, numba._dispatcher, numba.core.runtime._nrt_python, numba.np.ufunc._internal, numba.experimental.jitclass._box, skimage.measure._ccomp, skimage.measure._moments_cy, skimage.measure._find_contours_cy, skimage.measure._marching_cubes_lewiner_cy, PIL._imagingcms, msgpack._cmsgpack, xxhash._xxhash, cuda_utils, __triton_launcher (total: 272)

**Comment by Tavris1** [comment]
I suspect there may be a compatibility issue with the new SageAttention 2.2.0 wheel.
It turns out I can't attach the `.bat` file directly, so please do the following:

1. In the `Add-ons` folder, rename the existing `SageAttention-Multi (v2.2.0 and v3).bat` to `SageAttention-Multi (v2.2.0 and v3).bat.bak` (add .bak to the end).
2. Save the attached file to the same `Add-ons` folder.
3. Rename `SageAttention-Multi (v2.2.0 and v3).bat.txt` to `SageAttention-Multi (v2.2.0 and v3).bat` (just remove the `.txt` extension at the end). 
4. Run it again and see if the issue is resolved.

[SageAttention-Multi (v2.2.0 and v3).bat.txt](https://github.com/user-attachments/files/30631491/SageAttention-Multi.v2.2.0.and.v3.bat.txt)

**Comment by Hermit6202** [comment]
> I suspect there may be a compatibility issue with the new SageAttention 2.2.0 wheel. It turns out I can't attach the `.bat` file directly, so please do the following:
> 
> 1. In the `Add-ons` folder, rename the existing `SageAttention-Multi (v2.2.0 and v3).bat` to `SageAttention-Multi (v2.2.0 and v3).bat.bak` (add .bak to the end).
> 2. Save the attached file to the same `Add-ons` folder.
> 3. Rename `SageAttention-Multi (v2.2.0 and v3).bat.txt` to `SageAttention-Multi (v2.2.0 and v3).bat` (just remove the `.txt` extension at the end).
> 4. Run it again and see if the issue is resolved.
> 
> [SageAttention-Multi (v2.2.0 and v3).bat.txt](https://github.com/user-attachments/files/30631491/SageAttention-Multi.v2.2.0.and.v3.bat.txt)

Thank you. Worked for me. Thumbs up!

**Comment by Tavris1** [comment]
I think it will be fine now.
Please use `Update Easy-Install` and test.

**Comment by 4rc4nux** [comment]
@Tavris1 
The workaround did the job.

I see you already pushed a patch. Nice!
I have to delete the wheels before updating to 3.12.1 or it doesn't matter?

**Comment by Tavris1** [comment]
No need. Run `Update Easy-Install` and then install SageAttention again.
